index.html

```diff
@@ -27,22 +29,248 @@
                 <h1 class="intro-text">I’m Mariane
                     Glad to see you!</h1>
-                <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my
+                <p class="description">I’m a software developer! I can help you build a product , feature or website
+                    Look through some of my
                     work and experience! If you like what you see and have a project you need coded, don’t hestiate to
                     contact me.</p>
-            </section>
 
-            <section>
                 <a href="#" class="subtitle">LET’S CONNECT</a>
                 <ul class="links-group">
-                    <li><a href="#" class="link"><img src="img/Vectortwitter.svg" alt="twitter" class="img-link"></a></li>
-                    <li><a href="#" class="link"><img src="img/Linkedin iconlinkedin.svg" alt="Linkedin" class="img-link"></a></li>
+                    <li><a href="#" class="link"><img src="img/Vectortwitter.svg" alt="twitter" class="img-link"></a>
+                    </li>
+                    <li><a href="#" class="link"><img src="img/Linkedin iconlinkedin.svg" alt="Linkedin"
+                                class="img-link"></a></li>
+                    <li><a href="#" class="link"><img src="img/Vectorm.svg" alt="M" class="img-link"></a></li>
+                    <li><a href="#" class="link"><img src="img/Vectorgithub.svg" alt="github" class="img-link"></a></li>
+                    <li><a href="#" class="link"><img src="img/Vectorp-l.svg" alt="Peace & Love" class="img-link"></a>
+                    </li>
+                </ul>
+            </section>
+            <!-- works start here -->
+            <section class="works-section">
+                <!-- first card -->
+                <div class="cards-work">
+                    <img src="img/tonic-work.jpg" alt="project" class="cards-img">
+                    <div>
+                        <h1 class="work-title">Tonic</h1>
+                        <div class="frame-2">
+                            <h5 class="frame-subtitle">CANOPI</h5>
+                            <div class="rounded-bullet"></div>
+                            <h5 class="position-title">Back End Dev</h5>
+                            <div class="rounded-bullet"></div>
+                            <h5 class="work-year">2015</h5>
+                        </div>
+                        <p class="skills-description">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <ul class="skills-ul">
+                            <li class="skills-list">
+                                <div class="skills-container-1">
+                                    <h4 class="skills-item">html</h4>
+                                </div>
+                            </li>
```

Comment:
It is not required but maybe you can use a meaningful class name. For example, If someone sees skills-list, he should expect to have a list inside the class, but you using a single element as a list. Maybe you can use meaningful class names like below if you like or anything else: 
```
<li class="skills-list-item">
     <div class="skills-container-1">
            <h4 class="skills-item-header">html</h4>
     </div>
 </li>
```